lib/src/object_id.rs

```diff
@@ -103,14 +103,19 @@ macro_rules! impl_id_type {
 
 pub(crate) use {id_type, impl_id_type};
 
+/// Stores bytes representing the minimum length unique hexadecimal prefix of an
+/// identifier.
```

Comment:
I'm not sure we actually use it for converting a full id to a prefix. I know we use it for the reverse. So the type is used more generally to represent a hex prefix whether it has the minimum length or not.

Reply:
I reworded this completely. Let me know if it's better.